lib/traject/penn_near_east_config.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Numismatics CSV Mapping Configuration
+
+require_relative 'csv_reader'
+require_relative 'dlme_json_resource_writer'
+require_relative 'macros/dlme'
+require_relative 'macros/csv'
+extend Macros::DLME
+extend Macros::Csv
+
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'CsvReader'
+end
+
+to_field 'agg_provider', provider
+
+# MET Museum
+to_field 'id', normalize_prefixed_id('emuIRN')
+
+to_field 'cho_provenance', column('accession_credit_line')
+to_field 'cho_creator', column('creator')
+to_field 'cho_coverage', column('culture', split: '|')
+to_field 'cho_coverage', column('culture_area')
+to_field 'agg_data_provider', normalize_penn_egyptian_provider
```

Comment:
Do you want to rename this macro now that it's used in near east too?

Reply:
We can. I'm actually refactoring this in a separate branch to keep it simple, but a rename may be in order for now.

Reply:
Done in next update.